.bazelrc

```diff
@@ -3,6 +3,9 @@ startup --windows_enable_symlinks
 build --enable_runfiles
 test --test_output=errors
 
+# So that the bazel-bin folder is populated since we use transitions heavily
+--use_top_level_targets_for_symlinks
```

Comment:
Is this recommended for any rules_dotnet bazel repo? I don't really understand the reason yet.